make/migrations/postgresql/0040_2.1.0_schema.up.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE IF NOT EXISTS execution (
+    id SERIAL NOT NULL,
+    type varchar(16) NOT NULL,
+    status varchar(16),
+    status_message text,
+    trigger varchar(16) NOT NULL,
+    extra_attrs JSONB,
+    start_time timestamp DEFAULT CURRENT_TIMESTAMP,
+    end_time timestamp NULL,
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS task (
+    id SERIAL PRIMARY KEY NOT NULL,
+    execution_id int NOT NULL,
+    job_id varchar(64),
+    status varchar(16) NOT NULL,
+    status_code int NOT NULL,
+    status_revision int,
+    status_message text,
+    retry_count int,
```

Comment:
What does `retry_count` use?

Reply:
Some jobs may retry, use this field to record the retry times